zanata-war/src/main/resources/messages.properties

```diff
@@ -1178,18 +1177,18 @@ jsf.RequestJoinGroup=Request to add project version to group
 jsf.AlreadyInGroup=Already in Group
 jsf.email.joingrouprequest.AdditionalInfoMessage=To ensure your request is processed without delay, please provide any additional information that will help the group maintainers to process your request.
 !FIXME make key similar to related keys
-jsf.RequestToJoinLanguageTeamTitle=Request To Join "{0}" Language Team
-jsf.RequestRoleLanguageTeamTitle=Request Role in "{0}" Language Team
-
+jsf.email.RequestToJoinLanguageTeam.title=Request to join {0} [{1}] language team
 jsf.email.JoinGroupRequest.Subject=Request to join group "{0}"
-jsf.email.joinrequest.Subject=User "{0}" wants to join the "{1}" language team
-jsf.email.rolerequest.Subject=User "{0}" request for additional role in "{1}" language team
-jsf.email.AdditionalInfoMessage=To ensure your request is processed without delay, please provide any additional information that will help the team coordinators identify you and process your request.
-# {0} = locale display name
-jsf.email.ContactCoordinatorTitle=Contact "{0}" Coordinator
-# {0} = locale ID, {1} = locale native name
-jsf.contactLanguageTeamCoordinatorForLocale=Contact "{0} ({1})" Language Team Coordinators
+jsf.language.email.joinrequest.Subject=User:{0} request to join [{1}] language team
+jsf.language.email.AdditionalInfoMessage=Please provide any additional information that will help the team coordinators identify you and process your request.
+jsf.group.email.AdditionalInfoMessage=Please provide any additional information for group maintainer.
```

Comment:
Suggest: `Please provide any additional information that will help the group maintainers identify your project and process your request.`